consensus/module.go

```diff
@@ -65,7 +65,7 @@ type ConsensusModule struct {
 	MaxBlockBytes uint64
 }
 
-func Create(configPath, genesisPath string, useRandomPK bool) (modules.ConsensusModule, error) {
+func Create(configPath, genesisPath string, useRandomPK bool, configOptions ...func(*typesCons.ConsensusConfig)) (modules.ConsensusModule, error) {
```

Comment:
This is a great workaround. Depending on the outcome of https://github.com/pokt-network/pocket/discussions/234, we may or may not be able to remove it.

However, I still think we can merge this in and then update the configs across the whole DB subsequently.

Reply:
Maybe addressed by #235